tests/console/yast2_dns_server.pm

```diff
@@ -71,7 +71,7 @@ sub run {
     assert_screen 'yast2-dns-server-step3';
     # Enable dns server and finish
     send_key 'alt-s';
-    send_key 'alt-f';
+    wait_screen_change { send_key 'alt-f'; };
```

Comment:
not really necessary but you can also leave out the semicolon within the '{}' -> `wait_screen_change { send_key 'alt-f' };`